templates/faraday_rest.md

```diff
@@ -18,7 +18,9 @@ can be found here:
 {% endfor %}
 
 **NOTE**: The `byte` field type must be set as the base64 encoded string
-representation of a raw byte array.
+representation of a raw byte array. Also, any time this must be used in a URL path
```

Comment:
This is sometimes called "URL safe base64 encoding". Maybe we should mention that so people can find that more easily in the programming language they use?

Reply:
https://tools.ietf.org/html/rfc4648#section-5

Reply:
I'll add this link in.

Reply:
ok, done.